docs/build.sh

```diff
@@ -112,8 +112,8 @@ generate_api_rst v3
 
 # Fixup anchors and references in v3 so they form a distinct namespace.
 # TODO(htuch): Do this in protodoc generation in the future.
-find "${GENERATED_RST_DIR}"/api-v3 -name "*.rst" -print0 | xargs -0 sed -i "s#envoy_api_#envoy_v3_api_#g"
-find "${GENERATED_RST_DIR}"/api-v3 -name "*.rst" -print0 | xargs -0 sed -i "s#config_resource_monitors#v3_config_resource_monitors#g"
+find "${GENERATED_RST_DIR}"/api-v3 -name "*.rst" -print0 | xargs -0 sed -i -e "s#envoy_api_#envoy_v3_api_#g"
+find "${GENERATED_RST_DIR}"/api-v3 -name "*.rst" -print0 | xargs -0 sed -i -e "s#config_resource_monitors#v3_config_resource_monitors#g"
```

Comment:
I'm not a sed expert but I don't understand why `-e` is needed here and not above?

Reply:
Oops sorry nevermind was not reading correctly.